WikiWatershed/model-my-watershed
# HydroShare Export Backend Refactor

## Overview

Refactors HydroShare backend, grouping the models, serializers, and services under `export/hydroshare`. Also updates migrations. Also adds to `HydroShareClient.add_files` the ability to handle file-like objects.

Connects #2584 

## Testing Instructions

* Check out this branch
* Run migrations

      $ ./scripts/manage.sh migrate

* Export a project to HydroShare and ensure it works correctly
